Cargo.toml

```diff
@@ -14,17 +14,13 @@ readme = "README.md"
 keywords = ["selenium", "webdriver", "chromedriver", "geckodriver", "automation"]
 categories = ["api-bindings", "development-tools::testing"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-reqwest = { version = "0.10.0-alpha.2", features = ["blocking", "json"] }
+reqwest = { git = "https://github.com/seanmonstar/reqwest", features = ["blocking", "json"] }
```

Comment:
I'm wary of depending on something that may/will change. This breaks reproducible builds.

What if we lock the version (as it was before) and then release a new version of `thirtyfour` once the dependencies have released updated versions?

Reply:
Looks like reqwest 0.10.0 will be released soon so we should be able to just bump the version number then.
Does that work for you?